ui/src/main/webapp/less/configuration/style.less

```diff
@@ -92,3 +85,9 @@
     padding: 5px;
     text-decoration: none;
 }
+
+.metatype input:not([type="checkbox"]),
```

Comment:
Wasn't sure which less file to put this in, so let me know if you have a better location in mind.